Gemfile.tt

```diff
@@ -4,35 +4,54 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 ruby File.read(".ruby-version")
 
 gem "rails", "<%= Rails.version %>"
-gem "puma", "~> 4.1"
-gem "pg"<%= gemfile_requirement("pg") || ', "~> 0.18"' %>
+gem "puma"
+gem "pg"
 gem 'dotenv-rails', require: "dotenv/rails-now"
-gem "bootsnap"<%= gemfile_requirement("bootsnap") %>, require: false
-gem "sassc-rails"
-<% if gemfile_requirement("turbolinks") -%>
-gem "turbolinks"<%= gemfile_requirement("turbolinks") %>
-<% end -%>
-gem "webpacker"<%= gemfile_requirement("webpacker") %>
+gem "bootsnap", require: false
+
+gem "webpacker"
 gem "lograge"
 gem "okcomputer"
 gem "sentry-ruby"
 gem "sentry-rails"
 
-gem "rack-canonical-host", "~> 0.2.3"
+gem "rack-canonical-host"
+
+# The original asset pipeline for Rails [https://github.com/rails/sprockets-rails]
+gem "sprockets-rails"
+
+# Hotwire's SPA-like page accelerator [https://turbo.hotwired.dev]
+gem "turbo-rails"
+
+# Hotwire's modest JavaScript framework [https://stimulus.hotwired.dev]
+gem "stimulus-rails"
+
+# Use Redis adapter to run Action Cable in production
+# gem "redis"
+
+# Use Kredis to get higher-level data types in Redis [https://github.com/rails/kredis]
+# gem "kredis"
+
+# Use Active Storage variants [https://guides.rubyonrails.org/active_storage_overview.html#transforming-images]
+# gem "image_processing", "~> 1.2"
```

Comment:
Defaults from the vanilla Rails7 `Gemfile` that @joshmcarthur and I think make sense for us.